### ISSUE TITLE
Refined storage deletes items

### ISSUE THREAD

**Issue by pixiepuff** [NillerMedDild/Enigmatica5#505]
**modpack version**
Version 5.5

**Issue**
When the server i am playing on restarts disks,patterns be upgrades gets deleted from their respective inventory slots.

**Comment by Darkere** [comment]
What dimension are you in?

**Comment by pixiepuff** [comment]
Overworld

**Comment by NielsPilgaard** [comment]
`restarts disks,patterns be upgrades` could you rephrase please?

**Comment by pixiepuff** [comment]
When the server restarts, all disks,  patterns and upgrades (speed, stack, etc.) gets deleted.

**Comment by NielsPilgaard** [comment]
Thanks for rephrasing :) that is one nasty bug. May we have the latest.log, from a time it happened please?
